packages/core/nuxt-module/lib/modules/performance.js

```diff
@@ -0,0 +1,31 @@
+const { merge } = require('lodash');
+
+function pushScripts() {
+  this.options.render = merge(this.options.render, {
+    http2: {
+      push: true,
+      pushAssets: (request, response, publicPath, preloadFiles) => {
+        return preloadFiles
+          .filter(({ asType }) => asType === 'script')
+          .map(({ file, asType }) => `<${publicPath}${file}>; rel=preload; as=${asType}`);
+      }
+    }
+  });
+}
+
+function loadPurgeCss(options) {
+  // PurgeCSS module should be installed after all other modules
+  this.nuxt.hook('modules:done', moduleContainer => moduleContainer.addModule([ 'nuxt-purgecss', options ]));
```

Comment:
if we are adding other modules we should definitely prompt about this

Reply:
`nuxt-purgecss` module already logs information to the console by itself.